src/pages/teams/terms.astro

```diff
@@ -0,0 +1,1214 @@
+---
+import Layout from "../../layouts/Layout.astro";
+---
+
+<Layout title="Privacy Policy">
+  <section id="intro">
+    <div class="m-auto max-w-7xl px-4 pt-10">
+      <div
+        class="m-auto max-w-4xl mb-20 bg-slate-50 rounded-xl px-5 sm:px-10 md:px-20 py-10"
+      >
+        <h1 class="text-gray-900 text-2xl md:text-3xl font-semibold">
+          Terms of Service
+        </h1>
+        <div class="text-gray-700 text-left space-y-7">
+          <div class="space-y-3">
+            <p class="mt-2 font-bold text-sm">Last Updated: April 09, 2024</p>
+            <p class="italic">
+              <span class="font-semibold">Note:</span> These Terms of Service apply
```

Comment:
@josevalim, the version I shared with you before didn't have that "Note." That part makes it explicit that these TOS are related only to Livebook Teams, not Livebook.